dev-scheme/guile-ncurses/Manifest

```diff
@@ -1,2 +1 @@
 DIST guile-ncurses-1.6.tar.gz 783803 SHA256 e358791e2c8512ae0260f7e499cceddf364b50165ae4410aef3cc4ff0332b372 SHA512 992ab15799d8ae7b5a27dbfff8f96e8bb10ce23c88ae5a1c44364071f1d7f96aa8d90ed50875ea43284540d5cf3d5944092fb70bf48b5c7dfba5ebf47fc67f8e WHIRLPOOL bf666c71286da6569b8adce8e8cb0dd709d7a2a0fa67f5ba26f3ac8fd936485735bba73c2b25a48c8967c5ce82e667828b608c12eeca13d6711e4db131f05140
-EBUILD guile-ncurses-1.6.ebuild 716 SHA256 38ee6fd420f5bca25f24c98c184a4dee737922d351153160fed3b4109bdf143b SHA512 14df6f404447b8ccc61d869f8581dbbaac71ddbeb873966625707d27437273c3cb7369608b84eef62027a02f4e0885449723795170c338c5ea81e33f872d7f81 WHIRLPOOL fe80c14308fb261c7154dc0367f4a247188151cffc923e3edddfe6de216b5cfedf00d1940e73fd998fc9964cfbf1f4622c0bf3f37b90c4ac03b19c6091c342eb
```

Comment:
@pkulev nope, you had it enabled. I've made another commit so that your overlay makes better use of them. For more info, see [this page (russian)](https://www.linux.org.ru/forum/development/12474486?lastmod=1459416225099#comment-12479280).